src/main/java/taskat2/Women.java

```diff
@@ -0,0 +1,23 @@
+package taskat2;
+
+public class Women extends Human{
+
+    private int shoesAmount;
+
+    public Women(String name, String eyeColor, int age, int height, int shoesAmount) {
+        super(name, eyeColor, age, height);
+        this.shoesAmount = shoesAmount;
+    }
+
+    protected void shopping(){
+        System.out.println("Only women can go shopping!");
+    }
```

Comment:
if this method has same logic for ALL created Women instances, maybe this behaviour should belong to the CLASS, but not to INSTANCE? :)